### ISSUE TITLE
Stopwatch crashes immediately upon launch

### ISSUE THREAD

**Issue by deisenberg2** [carstengehling/jirastopwatch#20]
Hi... I've just installed the latest version of Stopwatch. I'm running Windows 7 Professional. I can't even get as far as the setup... the program simply crashes with no indication of what's happened. In case it's helpful, I'm attaching the Windows .wer file containing some info which might be useful.

Any guidance you can provide would be greatly appreciated... thank you!

David
[Report.wer.txt](https://github.com/carstengehling/jirastopwatch/files/286606/Report.wer.txt)

**Comment by carstengehling** [comment]
Stopwatch REQUIRES .NET 4.5 installed. Can you check if you have that on your machine?

**Comment by deisenberg2** [comment]
Actually, I have 4.6 installed.

**Comment by deisenberg2** [comment]
Carsten, I just uninstalled .NET 4.6 and installed 4.5, but I'm still having the same issue. I figured I'd let you know. Thanks, David

**Comment by carstengehling** [comment]
I am no .NET expert, but ikke believe, that you need 4.5 runtime separately installer, even if you already have 4.6

**Comment by deisenberg2** [comment]
Well, I seem to get the same result no matter which version of .NET I am running. It simply doesn't launch (and it produces the error log I attached) with 4.5, or 4.6.

**Comment by carstengehling** [comment]
I've made an experimental version based on 1.7.0, that tries to log all unhandled exceptions. You can download it here: http://gehling.dk/JiraStopWatch-1.7.0-errortrace.zip

Could you please try this, and then send me the logfile(s), that are generated? TIA

**Comment by deisenberg2** [comment]
Carsten,

Here you go; thank you!

David

**Comment by deisenberg2** [comment]
Carsten,

I think you got this via e-mail, but just in case... thank you!

David

![cid_ecfae97b250d48babe2dfb6007724b25 na_rtdom](https://cloud.githubusercontent.com/assets/19610416/16675213/ef23b1aa-448c-11e6-9451-8209098a050c.png)

[errortrace.log.txt](https://github.com/carstengehling/jirastopwatch/files/353371/errortrace.log.txt)

**Comment by carstengehling** [comment]
Thank you - I think I've found the problem. Could you try this (again experimental) version and see if it solves the problem?

http://gehling.dk/JiraStopWatch-1.7.0-fix1.zip

**Comment by deisenberg2** [comment]
Carsten,

That fixed it! Thank you!!

I should mention that the filters don't seem to be working in this version. I know that I'm successfully connected to our JIRA database, because if I manually type in a JIRA ticket number, StopWatch does successfully look up and display the ticket description (see the screenshot). But the Filter dropdown list is empty; no filters are present in the list. This is not a show-stopper, but I wanted to mention it.

David

![image](https://cloud.githubusercontent.com/assets/19610416/16711672/745d4f30-4635-11e6-837e-374425e62bae.png)

**Comment by carstengehling** [comment]
Good news! I'll make an official release this week then containing this bugfix.

I can see that you edited your post. You originally wrote something else about filter-loading. Is that resolved?

Also the screenshot looked weird in the bottom of the window - I would like to address that too. Are you running any kind of zoom-level on your Windows (and what Windows version is it?)

**Comment by deisenberg2** [comment]
Carsten,

Yes, I did edit my original post, in a futile attempt to cover my tracks. There is nothing wrong with the filter-loading; it worked as soon as I was clever enough to add my first filter. Sorry for the false alarm!

However, you are correct that there is something odd about the display. I'm running Windows 7 Professional. In "Control Panel... Display" I always have the "Make it easier to read what's on your screen" button set to Medium (125%). That makes StopWatch look like this:

![stopwatch at 125](https://cloud.githubusercontent.com/assets/19610416/16713612/f680f9b2-467b-11e6-8fde-7ad48296a82b.PNG)

As an experiment, I changed my display to 100%, and sure enough, that fixed the StopWatch display:

![stopwatch at 100](https://cloud.githubusercontent.com/assets/19610416/16713624/12c22678-467c-11e6-8d8e-c5840f84dd05.PNG)

Leaving the display at 100% isn't really an option for my middle-aged eyes. I will have to set the display at 125%. The StopWatch text does look a little strange that way, but it's certainly still usable.

David
